### PR TITLE
fix: typo

### PR DESCRIPTION
**Pull request by emergenz**
mising -> missing
-----
- [X] I searched the issue tracker and this hasn't been PRed before.
- [ ] My changes are not on [the do-not-PR list](https://doomemacs.org/d/do-not-pr) for this project.
- [X] My commits conform to [the git conventions](https://doomemacs.org/d/git-conventions).
- [ ] My changes are visual; I've included before and after screenshots.
- [ ] I am blindly checking these off.
- [X] Any relevant issues or PRs have been linked to.
- [ ] This a draft PR; I need more time to finish it.

### CODE REVIEW

**Comment by knl** [comment]
I think this is on do-not-PR list: https://discourse.doomemacs.org/t/do-not-pr/2406#all-projects-1

> Fewer than 3 trivial corrections per PR. This includes minor spelling, grammar, or syntax errors in docs, comments, or output messages that have no user-facing effect

**Comment by emergenz** [comment]
Okay, fair, sorry.

**Comment by emergenz** [comment]
One just needs to click on _merge_ though ;)

**Comment by dschrempf** [comment]
I agree with @emergenz :smile:

**Comment by knl** [comment]
> I agree with @emergenz 😄

Me too, it's a straightforward PR.

I'm just pointing that `My changes are not on [the do-not-PR list](https://doomemacs.org/d/do-not-pr) for this project.` doesn't match what's in that document.

**Comment by hlissner** [comment]
Thank you for bringing it to my attention, but PRs like these are on the DNP list for good reason. They are deceptively high-effort to process, and rarely arrive merge-ready (including this one, whose commit message uses the wrong type and needed a scope, i.e. `docs(python): ...`). People also get (understandably) frustrated having to fix these issues, and it's a honey pot for less experienced devs, who will need extra guidance to make those corrections.

TL;DR: Yours is not an egregious case by any means, but please refrain from making PRs like this in the future.

**Comment by emergenz** [comment]
After scrolling through the list of PRs, I get what you mean. Sorry for the inconvenience.
